client/src/featureform/providers/filestore.py

```diff
@@ -0,0 +1,139 @@
+import pandas as pd
+import os
+from ..register import ScalarType
+from .online_store import OnlineStore, OnlineTable
+
+
+class LocalFileTable(OnlineTable):
+    def __init__(self, name, variant, root_path, stype):
+        self.name = name
+        self.variant = variant
+        self.root_path = root_path
+        self.dir = f"{root_path}/{name}/"
```

Comment:
should we use pathlib so we don't have to deal with pathing stuff on our own? i.e. someone passes root path with a ton of backslashes or something?

Reply:
All the pathing stuff needs to change, this is temporary bc i need to interface merged